src/modules/countComments.js

```diff
@@ -0,0 +1,3 @@
+const totalComments = (commentsArray) => commentsArray.length;
+
+module.exports = totalComments;
```

Comment:
[OPTIONAL] Unfortunately, the previous reviewer didn't notice. So I will ask for it as optional as well. We need to use ES6 syntax and modules since it is one of the requirement of this project. Instead of module.exports you can use export default syntax like you used on other modules already.